server/test/services/export/JsonExporterTest.java

```diff
@@ -304,7 +310,12 @@ public void export_whenCheckboxQuestionIsNotAnswered_valueInResponseIsEmptyArray
             SubmittedApplicationFilter.EMPTY);
     ResultAsserter resultAsserter = new ResultAsserter(resultJsonString);
 
-    resultAsserter.assertValueAtApplicationPath(".kitchen_tools.selections", ImmutableList.of());
+    resultAsserter.assertJsonAtApplicationPath(
```

Comment:
**suggestion:** Another option so you don't have to worry as much about formatting would be to have `resultAsserter.assertJsonAtApplicationPath(".kitchen_tools.selections"..`
and
`resultAsserter.assertJsonAtApplicationPath(".kitchen_tools.question_type"..`

Here and in other parts of the file. Fine with either though.

Reply:
yeah... I wanted to start enforcing that other stuff we're not expecting doesn't show up in the API response, not just that the things we're expecting are there. But I don't like that the order matters in these asserts, and that the test failure .diffs are really hard to read

I'm planning to look into jayway.JsonAssert, which I think could be used here to make these more readable while also checking that we're not including anything we don't want to.

Reply:
Sounds good, makes sense!